dll/ntdll/ldr/ldrinit.c

```diff
@@ -598,8 +606,16 @@ LdrpInitializeThread(IN PCONTEXT Context)
         RtlActivateActivationContextUnsafeFast(&ActCtx,
                                                LdrpImageEntry->EntryPointActivationContext);
 
-        /* Do TLS callbacks */
-        LdrpCallTlsInitializers(Peb->ImageBaseAddress, DLL_THREAD_ATTACH);
+        _SEH2_TRY
+        {
+            /* Do TLS callbacks */
+            LdrpCallTlsInitializers(Peb->ImageBaseAddress, DLL_THREAD_ATTACH);
```

Comment:
This function already uses SEH for everything except RtlImageDirectoryEntryToData. Is it necessary to protect that?

Reply:
I wrapped that because I already wrapped a few ones together with the other one,
any idea how to test if we should or should not do it?
Or shall I just drop it here.